_docs-v3/event-data/events-json-feed.md

```diff
@@ -2,29 +2,73 @@
 title: events (as a json feed)
 ---
 
-A URL of a JSON feed that the calendar will fetch [Event Objects](event-object) from.
-
-FullCalendar will visit the URL whenever it needs new event data. This happens when the user clicks prev/next or changes views. FullCalendar will determine the date-range it needs events for and will pass that information along in GET parameters.
+In order for FullCalendar to fetch [Event Objects](event-object), it will need to visit the URL of the JSON Feed containing those events. This action occurs when the user clicks prev/next or changes calendar views. FullCalendar will determine the date-range it needs events for and will pass that information along in GET parameters.
```

Comment:
the first sentence makes it sound like a JSON feed is the *only* way for a calendar to be populated with events, when in reality, they can use other techniques like an array or a function. this introductory sentence should serve to explain what a JSON feed is. maybe even link to more information about JSON.

a JSON feed "containing" those events... not the best word. it generates/serves/produces JSON data for events